zmsentities/tests/Zmsentities/ExchangeTest.php

```
@@ -114,6 +114,73 @@ public function testGroupedWithRequestsSum()
+            ],
+        ];
+
+        $result = $exchange->withWeightedAverageProcessingTime();
+          // (10 * 20 + 20 * 5 + 15 * 2) / (20 + 5 + 2)
+          // = 330 / 27
+          // = 12.22
+
+   
+        $this->assertSame(
+            12.22,
+            $result-data['average_processingtime_overall']
```

PHPStan (2.2.8)

[error] Constant data not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)